servo_control.py

```diff
@@ -4,6 +4,7 @@
 class Servo_Controller():
 
     def __init__(self, min, mid, max, bReverse):
```

Comment:
Use named params here and define default values to use as backup in case they're not supplied?